src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const pricePerDay = 40;
+  const fullPrice = days * pricePerDay;
+  const basicDiscount = 20;
+  const advancedDiscount = 50;
+
+  if (days >= 7) {
+    return fullPrice - advancedDiscount;
+  } else if (days >= 3) {
+    return fullPrice - basicDiscount;
+  }
```

Comment:
```suggestion
  if (days >= 7) {
    return fullPrice - advancedDiscount;
  }
  
  if (days >= 3) {
    return fullPrice - basicDiscount;
  }
```

Reply:
as far as `return` exits the program right away, you could simplify it

Reply:
Fixed. Thank you for the hint!